src/DesktopVirtualization/README.md

```diff
@@ -93,4 +93,11 @@ directive:
       verb: Get
       subject: ActiveApplication
     remove: true
+  - where:      
+      verb: Remove
+      parameter-name: Force
+      subject: Updated description for clarity.
```

Comment:
"subject" is part of the name of the cmdlet, and it's used to identify which cmdlet to update. "Updated description for clarity." isn't a valid subject. See [prefix, subject-prefix and subject](https://github.com/Azure/autorest.powershell/blob/main/docs/directives.md#prefix-subject-prefix-and-subject).

Once you've corrected the subject, like I said, run `autorest` and `./build-module.ps1` to regenerate the module. You would expect the documents in `docs/` folder to be updated according to the directive after this step, otherwise the directive is incorrect.

Reply:
@isra-fel I have changed the parameter as per your suggestions and verified with `autorest` and `build-module` and documents got generated correctly under docs.